src/main/java/com/epam/java/pdp/figures/Shape.java

```diff
@@ -44,24 +44,31 @@ void calculateLineLengths() {
 
     }
 
-    void arePointsOnTheSingleLine() {
+    boolean arePointsOnTheSingleLine() {
         calculateLineLengths();
         double maxLength = 0;
-//        for (int i=0; i< points.size(); i++){
-//            if (lines.get(i) > maxLength){
-//                maxLength = lines.get(i);
-//            }
-//        }
-        for (int i = 0; i < points.size(); i++) {
-//            int nextIndex = i + 1;
-            if (maxLength == lineLengths.get(i) + lineLengths.get(i++)) {
+
+        for (int i = 0; i < lineLengths.size(); i++) {
+            if (lineLengths.get(i) > maxLength) {
+                maxLength = lineLengths.get(i);
+            }
+        }
+        for (int i = 0; i < lineLengths.size(); i++) {
+            int nextIndex = i + 1;
+            if (nextIndex == lineLengths.size()) {
+                nextIndex = 0;
+            }
+            if (maxLength == lineLengths.get(i) + lineLengths.get(nextIndex)) {
                 System.out.println("Points are on the same line.");
+                oneLine = true;
             } else {
                 System.out.println("Points are not on the single line.");
+                oneLine = false;
+
             }
+
         }
-        for (Double maxLineLength : lineLengths) {
-        }
+        return oneLine;
```

Comment:
Instead of defining the instance attribute named oneLine please use the result of arePointsOnTheSingleLine method execution.